css/desktop.css

```diff
@@ -0,0 +1,341 @@
+@media screen and (min-width: 768px) {
+  body {
+    font-size: 5px;
+    max-width: 100%;
+  }
```

Comment:
As previously requested.

The homepage - desktop version still has a horizontal scroll between widths of `768px - 1098px`.

Give this a fix just like you did with the about page.